src/pages/organize/[orgId]/projects/calendar/index.tsx

```diff
@@ -25,17 +25,14 @@ const scaffoldOptions = {
 export const getServerSideProps: GetServerSideProps = scaffold(async (ctx) => {
   const { orgId } = ctx.params!;
 
-  await ctx.queryClient.prefetchQuery(
-    ['org', orgId],
-    getOrg(orgId as string, ctx.apiFetch)
+  const apiClient = new BackendApiClient(ctx.req.headers);
+  const organization = await apiClient.get<ZetkinOrganization>(
```

Comment:
If the organization does not exist, normal organizers will not even get this far because `scaffold()` already checks for access. But superusers will get this far, and will get a 500 when the API request throws an exception that is not currently being handled.